.circleci/config.yml

```diff
@@ -0,0 +1,59 @@
+version: 2.1
+#orbs:
+#  slack: circleci/slack@3.2.0
+executors:
+  daml_sdk:
+    docker:
+      - image: digitalasset/daml-sdk:0.13.54
```

Comment:
Should we be hardcoding the SDK version here? Wouldn't it be better to always run against latest and sound the claxon if the examples don't build with the latest?

Unless we keep all examples on the same SDK version, the CI will end up downloading multiple SDK versions anyway.

Reply:
On the one hand I think that running these tests should be a CI hook on the main `daml` repo, not here. On the other hand, within this repo, it is also a good idea to also test against the latest release. But how do you do that? If we leave off the `0.13.54` will that automagically pull the latest?

Reply:
if the daml CI uses the `latest` tag we can use that. @garyverhaegen-da might have an idea

Reply:
We deliberately do not push a latest tag because the image is targeted at CI usage and we believe CI should know what version it's testing against.